usaspending_api/search/v2/urls_search.py

```diff
@@ -1,12 +1,16 @@
 from django.conf.urls import url
-from usaspending_api.search.v2.views.search import SpendingOverTimeVisualizationViewSet, \
-    SpendingByCategoryVisualizationViewSet, SpendingByGeographyVisualizationViewSet, SpendingByAwardVisualizationViewSet,\
-    SpendingByAwardCountVisualizationViewSet
+from usaspending_api.search.v2.views.search import SpendingByAwardCountVisualizationViewSet
```

Comment:
Yeah, it isn't very clean when a "single line" import statement no longer fits on one line, but there are other benefits:
* It does help with future PRs since any edits will only affect the specific module or class on that line.
* Readability
* Meets with [Google's style guide](https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting)